ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerFeatureLayer/README.metadata.json

```diff
@@ -7,6 +7,7 @@
         }
     ],
     "description": "Start a local feature service and display its features in a map.",
+    "featured": true,
```

Comment:
If a local server sample is featured but local server isn't supported, the sample doesn't show up in the featured list because the sample's json file is never read.